controllers/books.js

```diff
@@ -1,44 +1,66 @@
 import { v4 as uuidv4 } from 'uuid';
-let books = []
+import fs from 'fs';
 
 export const getBooks = (req, res) => {
-      res.send(books);
+	const books  = JSON.parse(fs.readFileSync('./data/books.json'));
+	res.send(books);
 }
 
 export const createBook = (req, res) => {
-      const book = req.body;
-      books.push({ ...book, id: uuidv4()});
-      res.send(`Book with the name ${book.bookTitle} added to the database!`);
-}
+	let books =  [...JSON.parse(fs.readFileSync('./data/books.json'))]
```

Comment:
I see that this `JSON.parse(fs.readFileSync('./data/books.json'))` is being used many places. You will probably not need this after implementing the module import suggestion. However, such reused code pieces can be declared at the top of the file and used in all the functions.